SMLHelper/Patchers/ItemActionPatcher.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Harmony;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace SMLHelper.V2.Patchers
+{
+    public class ItemActionPatcher
+    {
+        #region Internal Fields
+
+        internal const ItemAction MiddleClickItemAction = (ItemAction)1337;
```

Comment:
Can we leave a comment regarding with 1337 is?

Reply:
A custom constant value is required to differenciate between normal `ItemAction`s and the custom middle click one. This is checked in another patch. (I know that this is not the best way to handle this)

Reply:
When I said leave a comment, I mean in the source code.
Either a comment or `<summary>` documentation.

We don't need to change this, only leave notes so that we aren't left wondering why we did this months later.